tests/utils.py

```diff
@@ -0,0 +1,7 @@
+import aiohttp
+
+
+async def run_in_session(converter, method, args):
```

Comment:
this method is not running things, please consider renaming it

Reply:
Done in f7c6ac4